k8s/aat/common/ccpay/ccpay-payment-app.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: ccpay
+  namespace: fees-pay
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: ccpay
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: payment-api
+    version: 0.1.0
+  values:
+    rollback:
+      enable: true
+    java:
+      replicas: 2
+      image: ${IMAGE_NAME}
```

Comment:
How will it find this ${IMAGE_NAME} ?

Reply:
If you see the file https://github.com/hmcts/cnp-flux-config/blob/master/k8s/aat/common/ccpay/ccpay-bulkscanning-app.yaml you will get to know what needs to be done for image name.

`hmctspublic.azurecr.io/payment/api:prod-718aea76`